tests/numpy/test_np_reduction.py

```diff
@@ -91,4 +91,4 @@ def test_argops(nps_app_inst):
     nums.core.settings.system_name = "serial"
     nps_app_inst = application_manager.instance()
     test_reductions(nps_app_inst)
-    test_argops(nps_app_inst)
+    # test_argops(nps_app_inst)
```

Comment:
No `test_argops` in this module.

Reply:
Are you saying you'd like to remove the function and the call in main altogether, or to uncomment this line?

Reply:
I misread this diff! I thought it was part of the new test module you created for nans. Please disregard. This can remain as-is.